scripts/gitpod_api_wizard.sh

```
@@ -34,8 +34,8 @@ function main() {
 		}
 		log info "Writing API_KEY to current workspace($GITPOD_WORKSPACE_ID) filesystem";
 		write_api_key "$api_key";
-		printf '\n\n>> %s' "API key setup complete! Press any key to contine ..." && read;
+		printf '\n\n>> %s' "API key setup complete! Press any key to continue ..." && read;
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)